include/swift/Serialization/ModuleFormat.h

```diff
@@ -344,6 +344,18 @@ enum SpecialModuleID : uint8_t {
   NUM_SPECIAL_MODULES
 };
 
+// Continuation of the special module IDs above for special names that don't
+// have an identifier. As above, these IDs must \em not be renumbered or
+// reordered without incrementing VERSION_MAJOR.
+enum SpecialNameID : uint8_t {
+  SUBSCRIPT_ID = NUM_SPECIAL_MODULES,
```

Comment:
Yeah, I think folding the two enums together would be the right way to go. Didn't really feel too comfortable with it but wanted an opinion on whether using the same enum for both special module names and other special names would be alright.